apps/src/dropletUtils.js

```diff
@@ -60,7 +60,7 @@ exports.getTime = function () {
 exports.promptNum = function (text) {
   var val;
   do {
-    val = parseInt(window.prompt(text), 10);
+    val = Number(window.prompt(text));
```

Comment:
Interesting. Thanks for sharing that link. I think you're right that probably either is fine. I'm going to stick with this since it's what I already have :)